spanner/read_test.go

```diff
@@ -1213,6 +1215,11 @@ func (sr *sReceiver) Recv() (*sppb.PartialResultSet, error) {
 	return sr.rpcReceiver.Recv()
 }
 
+func (sr *sReceiver) Context() context.Context {
+	return context.TODO()
```

Comment:
What does this TODO mean?

Reply:
This receiver is used in few of th unit tests, returning context.TODO here meant there are those tests nothing.